code/controllers/subsystem/economy.dm

```diff
@@ -41,6 +43,8 @@ SUBSYSTEM_DEF(economy)
 	var/budget_to_hand_out = round(budget_pool / department_accounts.len)
 	for(var/A in department_accounts)
 		new /datum/bank_account/department(A, budget_to_hand_out)
+	for(var/A in banker_accounts)
+		new /datum/bank_account/department(A, INFINITY)
```

Comment:
Yeah, I still need to test it but the BYOND pager is sending me dots instead of running the .dmb like it used to. INFINITY is just a define of a large number formatted with scientific notation so I expect it should work like a regular number.